service/storage_proxy.cc

```diff
@@ -482,8 +482,10 @@ class storage_proxy::remote {
                         errors.count++;
                         errors.local = replica::try_encode_replica_exception(eptr);
                         seastar::log_level l = seastar::log_level::warn;
-                        if (is_timeout_exception(eptr) || std::holds_alternative<replica::rate_limit_exception>(errors.local.reason)) {
-                            // ignore timeouts and rate limit exceptions so that logs are not flooded.
+                        if (is_timeout_exception(eptr)
+                            || std::holds_alternative<replica::rate_limit_exception>(errors.local.reason)
+                            || std::holds_alternative<abort_requested_exception>(errors.local.reason)) {
```

Comment:
The seastar coding style (which we follow) requires to add two indents when you are breaking a long line like this. See: https://github.com/scylladb/seastar/blob/master/coding-style.md#long-lines.